src/components/typography/list.component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Typography from './typography.component';
+
+const getListStyleType = (as) => {
+  if (as === 'ul') {
+    return 'square';
+  }
+  return 'decimal';
+};
+
+const List = ({ children, as = 'ul', ...props }) => (
+  <Typography
+    variant='p'
+    as={ as }
+    listStyleType={ getListStyleType(as) }
+    { ...props }
+  >{children}
+  </Typography>
+);
+
+List.propTypes = {
+  ...Typography.propTypes,
+  as: propTypes.oneOf(['ul', 'ol'])
+};
+
+const ListItem = ({ children, ...props }) => (
+  <Typography
+    as='li' variant='p'
```

Comment:
Really minor but I'd put these props on separate lines as above. I'll hurry up and get prettier in next pioneer Friday so we don't even have to think about this!